pkg/testutil/tempdir.go

```diff
@@ -53,9 +53,10 @@ func (td *TempDir) Root() string {
 // Path returns the path to a file in the temp directory.
 // The input file is expected to use '/' as directory separator regardless of the host OS.
 func (td *TempDir) Path(file string) string {
-	pathElems := []string{td.root}
-	pathElems = append(pathElems, strings.Split(file, "/")...)
-	return filepath.Join(pathElems...)
+	if strings.HasPrefix(file, td.root) {
```

Comment:
do you mind explaining why this case occurs?

Reply:
in `info_test.go:120`, I use the environment paths object to construct the desired destination path for a plugin receipt. This is already an absolute path into the tempdir. So when I want to create a file inside tempdir, I must not nest that into tempdir again.